### DISCUSSION TITLE
Images aren't adhering to prompts given.

### DISCUSSION THREAD

**Discussion by MancV21** [d8ahazard/sd_dreambooth_extension#550] [Q&A]
Hi.

I've trained a model using 19 images of Emilia Clarke as a way of testing training. It seems to work OK but when I enter prompts, it ignores what I ask. For example, once trained (only 800 steps for now, but it's enough for a likeness) if I enter

A **beautiful woman** with short blue hair, I get results like this > 
![tmpbw2dfvnf](https://user-images.githubusercontent.com/48100565/207909914-e27bba04-4936-4043-85c8-8fff8b8830f9.png)

However if I use the instance and class token **Emilia_Clarke woman** with short blue hair (I've used different tokens to check it's that at fault, for example ECW) it completely ignores the prompt of giving her short blue hair and just creates an image close to one of the trained images >
 ![tmpv9eibolx](https://user-images.githubusercontent.com/48100565/207910135-0b57d18e-1ee4-4256-8d1a-422d047b3c3c.png)

Every woman now looks like Emilia Clarke in the model too (I based it on the default 1.5 model). What am I doing wrong? I know I could use more training steps, but for now ignore that, there's no point in doing that until I know what I'm messing up in the first place. Thanks for your help.

Here are my settings >
![image](https://user-images.githubusercontent.com/48100565/207911596-9e328f8a-4a7b-47bb-aa5a-76b22bfbf8e4.png)

**Comment by zahdab** [comment]
I used to have this issue, only way around it for me was to start adding file word descriptions for each image, which brought back flexibility ... i think usually think is a symptom of over training a model

**Comment by nanafy** [comment]
You overtrained the data set, so now the instance prompt dominates all the other tokens and overpowers them. Zahdab is correct that it is a symptom over overfitting. The quickest way to fix this without retraining, is to switch around the position of your instance token. Place it at the end of your prompt.  

e.g 

detailed, intricate, perfect feminine face, photorealistic, photoshop, volumetric lighting of  Emilia_Clarke woman

And you're learning rate is insanely high. Try lowering it to 1.25e-6 or something if you aren't using any class images either.

**Reply by nanafy** [reply]
Reply here if you need more help!
